templates/ReactReduxSpa/tsconfig.json

```diff
@@ -5,7 +5,12 @@
     "jsx": "preserve",
     "experimentalDecorators": true,
     "sourceMap": true,
-    "skipDefaultLibCheck": true
+    "skipDefaultLibCheck": true,
+    "lib": ["es6", "dom"],
+    "typeRoots": [ "node_modules/@types" ],
```

Comment:
Not specifically, it's just nice to see at a glance what Type definitions are being used.